databricks/koalas/utils/accessors.py

```diff
@@ -0,0 +1,177 @@
+#
+# Copyright (C) 2019 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+Utility features. The features here are specific to Koalas.
+"""
+from collections import OrderedDict
+from typing import Tuple, Union, TYPE_CHECKING
+
+from databricks.koalas.internal import InternalFrame, SPARK_INDEX_NAME_FORMAT
+from databricks.koalas.utils import name_like_string, scol_for
+
+if TYPE_CHECKING:
+    from databricks.koalas.frame import DataFrame
+
+
+class UtilsFrameMethods(object):
+    """ Utility features for DataFrame. Usually, The features here are specific to Koalas. """
+
+    def __init__(self, frame: "DataFrame"):
+        self._kdf = frame
+
+    def attach_sequence_column(self, column: Union[str, Tuple[str, ...]]) -> "DataFrame":
+        """
+        Attach a column of a sequence that increases one by one.
+
+        .. note:: this uses Spark's Window without specifying partition specification.
```

Comment:
nit: Maybe `this uses Spark's Window without specifying partition.` or `this uses Spark's Window without  partition apecification.` is enough?
`specifying` and `specification` seems duplicated ?

Reply:
Actually I copied the note from another place. Maybe we should correct all of them in a separate PR if needed.

Reply:
@ueshin Oh I got it, I think It's not a big deal.Let's fix them in a separate PR later.